2.5/2.5.2/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.5.2
+Создайте 4 переменные, которые будут содержать имена собаки "Шарик", кота "Вася" и их возраст - 5 и 3.
+С помощью этих переменных отобразите 2 сообщения в консоли:
+
+У меня есть кот Вася и собака Шарик
+Коту Вася 3 года, а собаке Шарик - 5 лет. В сумме 8 лет.
+<script>
+	const dogName = 'Шарик';
+	const catName = 'Вася';
+	const dogAge = 5;
+	const catAge = 3;
+
+	alert(`У меня есть кот ${catName} и собака ${dogName}`);
```

Comment:
![image](https://github.com/tacya9/JS-1-2/assets/115877101/3b8477d9-6222-43f8-bf25-e531e3c5b14c)
not by alert